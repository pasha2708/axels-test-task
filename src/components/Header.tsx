import React from 'react';

import Title from '../styled/components/Header';

const Header = () => <Title>Test App</Title>;

export default Header;
