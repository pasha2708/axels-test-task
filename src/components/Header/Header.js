import React from "react";
import styled from 'styled-components'

const Title = styled.h1`
  text-align: center;
  font-family: 'Inter', sans-serif;
  font-size: 3em;
  text-transform: uppercase;
  margin: 20px 0;
`

function Header() {
  return (
    <Title>Test App</Title>
  );
}

export default Header;