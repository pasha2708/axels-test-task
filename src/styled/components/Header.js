import styled from 'styled-components';

const Title = styled.h1`
  text-align: center;
  font-family: "Inter", sans-serif;
  font-size: 3em;
  text-transform: uppercase;
  margin: 20px 0;
`;

export default Title;