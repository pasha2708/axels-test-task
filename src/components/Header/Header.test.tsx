import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe("Header component", () => {
  it("should render Header", () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
