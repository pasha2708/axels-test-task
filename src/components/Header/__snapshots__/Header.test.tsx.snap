// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header 1`] = `
<styled.h1>
  Test App
</styled.h1>
`;
